### PR TITLE
Changing uppercase elements into lowercase.

### PR DESCRIPTION
**Pull request by rnirmal**

### CODE REVIEW

**Comment by hub-cap** [comment]
Cool. Paul changed it in the example gen as well.

**Comment by rnirmal** [comment]
Yup but we didn't update the examples.

**Comment by TimSimpsonR** [comment]
+1

**Comment by cp16net** [comment]
Nice +1

**Comment by hub-cap** [comment]
+1

File: apidocs/src/resources/samples/db-list-users-response.xml [hunk 1]
```diff
@@ -1,33 +1,24 @@
 HTTP/1.1 200 OK
 Content-Type: application/xml
-Content-Length: 808
```
Comment (ed-) [inline apidocs/src/resources/samples/db-list-users-response.xml:3]: Nuthin' sounds quite like an eight 

oh

eight

**Comment by imsplitbit** [comment]
+1

File: apidocs/src/resources/samples/db-list-users-response.xml [hunk 2]
```diff
@@ -1,33 +1,24 @@
...
-		<databases>
-			<database>
-				<name>databaseA</name>
-			</database>
-			<database>
-				<name>databaseB</name>
-			</database>
-		</databases>
-	</user>
+    <user name="userwith2dbs">
```
Comment (ed-) [inline apidocs/src/resources/samples/db-list-users-response.xml:7]: Nitpick:

`userwith2dbs`
`userwith3db`

don't go holding this PR up for this though

**Comment by ed-** [comment]
+1

**Pull request merged**